dynamic-config/config-kie/src/main/java/org/apache/servicecomb/config/kie/client/KieClient.java

```diff
@@ -48,16 +49,27 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(KieClient.class);
 
-  private ScheduledExecutorService EXECUTOR = Executors.newScheduledThreadPool(1);
+  private ScheduledExecutorService EXECUTOR = Executors.newScheduledThreadPool(1, (r) -> {
+    Thread thread = new Thread(r);
+    thread.setName("org.apache.servicecomb.config.kie");
+    thread.setDaemon(true);
+    return thread;
+  });
 
   private static final long TIME_OUT = 10000;
 
+  private static AtomicBoolean IS_FIRST_PULL = new AtomicBoolean(true);
+
+  private static final int LONG_POLLING_WAIT_TIME = 30;
```

Comment:
long polling timeout is equals to client request timeout, will this cause client timeout frequently?

Reply:
After checking , vert.x default strategy is never timeout. I set the idle timeout 50s.